src/ralph/deployment/api.py

```diff
@@ -70,3 +78,69 @@ class Meta:
             timeframe=TIMEFRAME,
             expiration=EXPIRATION,
         )
+
+
+class IPAddressChange(object):
+
+    __slots__ = ['current_ip_address', 'new_ip_address']
```

Comment:
Do we really need to handle so much of these objects that we need slots? According to people smarter than me ``__slots__`` should be used only for performance reasons.

I believe namedtuple would suit better here:

    IPAddressChange = namedtuple(['current_ip_address', 'new_ip_address']